### ISSUE TITLE
isConstructor() error in Initializable.sol

### ISSUE THREAD

**Issue by elishadrion** [OpenZeppelin/openzeppelin-sdk#1440]
Compiling or using the [Initializable.sol](https://github.com/OpenZeppelin/openzeppelin-sdk/blob/master/packages/lib/contracts/Initializable.sol) isn't possible due to an error in the [isConstructor()](https://github.com/OpenZeppelin/openzeppelin-sdk/blob/master/packages/lib/contracts/Initializable.sol#L48) function. The function in question and the error are highlighted below ;

```javascript
  /// @dev Returns true if and only if the function is running in the constructor
  function isConstructor() private view returns (bool) {
    // extcodesize checks the size of the code stored in an address, and
    // address returns the current address. Since the code is still not
    // deployed when running a constructor, any checks on its code size will
    // yield zero, making it an effective way to detect if a contract is
    // under construction or not.
    uint256 cs;
    assembly { cs := extcodesize(address) } <--
    return cs == 0;
  }
```

This is invalid in Solidity as in `address` is a reserved keyword.

Given the context, it seems it should be ;

```javascript
    uint256 cs;
    address _addr = address(this);
    assembly { cs := extcodesize(_addr) }
    return cs == 0;
```

**Comment by frangio** [comment]
Thank you for reporting! This is being fixed in the 2.7 release.

**Comment by frangio** [comment]
It was fixed in #1413.
